core/src/main/java/org/mskcc/cbio/maf/MafUtil.java

```diff
@@ -208,79 +180,57 @@
-	private int oncoProteinPosEndBeIndex = -1;
-
-	// Mutation Assessor column indices
-	private int maFImpactIndex = -1; // MA:FImpact
-	private int maFisIndex = -1; // MA:FIS
-	private int maLinkVarIndex = -1; // MA:link.var
-	private int maLinkMsaIndex = -1; // MA:link.MSA
-	private int maLinkPdbIndex = -1; // MA:link.PDB
-	private int maProteinChangeIndex = -1; // MA:protein.change
-
-	// custom filtering of passenger and driver mutations column indices
-	private int driverIndex = -1; //cbp_driver
-	private int driverAnnIndex = -1; //cbp_driver_annotation
-	private int driverTiersIndex = -1; //cbp_driver_tiers
-	private int driverTiersAnnIndex = -1; //cbp_driver_tiers_annotation
-
-	// number of headers in the header line
+    private int oncoRefseqMrnaIdIndex = -1;
+    private int oncoUniprotNameIndex = -1;
+    private int oncoUniprotAccessionIndex = -1;
+    private int oncoCodonChangeIndex = -1;
+    private int oncoProteinPosStartIndex = -1;
+    private int oncoProteinPosEndIndex = -1;
+
+    // Mutation Assessor column indices
+    private int maFImpactIndex = -1; // MA:FImpact
+    private int maFisIndex = -1; // MA:FIS
+    private int maLinkVarIndex = -1; // MA:link.var
+    private int maLinkMsaIndex = -1; // MA:link.MSA
+    private int maLinkPdbIndex = -1; // MA:link.PDB
+
+    // custom filtering of passenger and driver mutations column indices
+    private int driverIndex = -1; //cbp_driver
+    private int driverAnnIndex = -1; //cbp_driver_annotation
+    private int driverTiersIndex = -1; //cbp_driver_tiers
+    private int driverTiersAnnIndex = -1; //cbp_driver_tiers_annotation
+
+    // number of headers in the header line
     private int headerCount;
 
-	// mapping for all column names (both standard and custom columns)
-	private HashMap<String, Integer> columnIndexMap;
+    // mapping for all column names (both standard and custom columns)
+    private HashMap<String, Integer> columnIndexMap;
 
-	/**
+    /**
      * Constructor.
      *
      * @param headerLine    Header Line.
      */
-    public MafUtil(String headerLine)
-    {
+    public MafUtil(String headerLine) {
         // init column index map
-	    this.columnIndexMap = new HashMap<String, Integer>();
+        this.columnIndexMap = new HashMap<String, Integer>();
 
         // split header names
-    	String parts[] = headerLine.split("\t");
+        String parts[] = headerLine.split("\t");
 
-    	// update header count
+        // update header count
         this.headerCount = parts.length;
 
         // find required header indices
-        for (int i=0; i<parts.length; i++)
-        {
+        for (int i=0; i<parts.length; i++) {
             String header = parts[i];
 
-	        // put the index to the map
-	        this.columnIndexMap.put(header.toLowerCase(), i);
+            // put the index to the map
+            this.columnIndexMap.put(header.toLowerCase(), i);
 
-	        // determine standard & default column indices
```

Comment:
side comment : This has runtime order(n^2) on the number of columns in the maf that we recognize. There is a better way to scan the column headers and determine the indexes. But even with 100 columns, it is only ~ 10000 string comparisons which will only waste something like 1 or 2 seconds per MAF